.github/workflows/tests.yml

```diff
@@ -92,3 +113,12 @@ jobs:
           status: ${{ job.status }}
           steps: ${{ toJson(steps) }}
         if: failure()
+
+  delete-wsi-artifacts:
```

Comment:
why do we need this?

Reply:
for a matter of fairness in favor of github. :D basically the artifacts have a retention time of 1 day, and i don't wanna have that useless thing that cannot be used outside each workflow, so each roud i'll delete it